test/integration/match_by_image_test.go

```diff
@@ -804,6 +804,12 @@ func TestMatchByImage(t *testing.T) {
 
 			// The single match in the actual results is the same in ignoredMatched
 			// but must the details of the VEX matcher appended
+			if len(vexedResults.Sorted()) < 1 {
```

Comment:
Added this check because of the naked `[0]` access below which was panic when we got no vexed results as a consequence of the PURL changes.